modules/preview-react/form-field/stories/examples/GroupedInputs.tsx

```diff
@@ -0,0 +1,216 @@
+import React from 'react';
+import {FormFieldGroup} from '@workday/canvas-kit-preview-react/form-field';
+import {PrimaryButton, SecondaryButton} from '@workday/canvas-kit-react/button';
+import {system} from '@workday/canvas-tokens-web';
+import {Banner} from '@workday/canvas-kit-react/banner';
+
+import {Checkbox} from '@workday/canvas-kit-react/checkbox';
+import {RadioGroup} from '../../../radio';
+import {createStyles} from '@workday/canvas-kit-styling';
+import {AriaLiveRegion} from '@workday/canvas-kit-react/common';
+
+const formStyles = createStyles({
+  marginBlockStart: system.space.x3,
+  marginBlockEnd: system.space.x3,
+});
```

Comment:
Is there margin elsewhere? If not, then this could be:
```suggestion
const formStyles = createStyles({
  margin: `${system.space.zero} ${system.space.x3}`,
});
```